image-types/rhel8/kvm-guest-image.md

```diff
@@ -1,7 +1,9 @@
 # KVM Guest Image
 
-This is an image meant to be used as a generic base image for various
-virtualization environments.
+This is an image meant to be used as a generic base image for KVM
+based hypervisors. It is mainly used with RHV or RHOSP. Using this
```

Comment:
I'd prefer spelling `RHV` and `RHOSP` out once, and linking to https://access.redhat.com/products/red-hat-virtualization and https://access.redhat.com/products/red-hat-openstack-platform (if these are the best product links), respectively.

Reply:
I also think we should at least once write the full name before we start using abbreviations. There is a lot of TLAs in IT world and way too many conflicting TLAs in RH portfolio. So I'd also mention what RHSM and RHUI stands for because every time I read it, I think about user interface :-).

(RHEL, CDN, and KVM might be fine without)

Reply:
Sure, makes sense. I'll write full names and add links to products. Thanks.

Reply:
I kept KVM and CDN as abbreviations, but other than these, I included also full names, together with links where appropriate. I also changed commit messages to use imperative.